src/components/ErrorBoundary/WithErrorBoundary.js

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
```

Comment:
I think if you use .js on this file instead of .ts it may skip the typescript compiler.

Reply:
It should be tsx extension , let js only for logical methods, If you render components or return htmlComponents  like the method withErrorBoundary you should use typescript.